### PR TITLE
7 create containers composition

### PR DESCRIPTION
**Pull request by YuryHouse**
feat docker-compose.yml, Dockerfile, env & fix package.json

### CODE REVIEW

File: .production.env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=7000
```
Comment (raman-davidovich) [inline .production.env:1]: Please add new line in the end of the file.

File: .development.env [hunk 2]
```diff
@@ -0,0 +1 @@
+PORT=5000
```
Comment (raman-davidovich) [inline .development.env:1]: Please add new line in the end of the file.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+COPY ./dist ./dist
+
+CMD ["yarn", "start:dev"]
```
Comment (raman-davidovich) [inline Dockerfile:13]: Please add new line in the end of the file.

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    env_file:
+      - .development.env
+    volumes:
+      - .:/app
+      - /app/node_modules
+    ports:
+      - 5000:5000
+      - 9229:9229
+    command: yarn start:dev
+    restart: always
```
Comment (raman-davidovich) [inline docker-compose.yml:17]: Please add new line in the end of the file.

File: .production.env (hunk 1 above)
Comment (YuryHouse) [reply .production.env]: All files were fixed

**Review by raman-davidovich — APPROVED** [review]

**Review by bakhromov96 — APPROVED** [review]
🚀

**Review by gitkoshelew — APPROVED** [review]

**Pull request merged**